src/script/lua_api/l_object.cpp

```diff
@@ -1063,6 +1063,9 @@ int ObjectRef::l_get_luaentity(lua_State *L)
 int ObjectRef::l_is_player_connected(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
+	// This method was once added for a bugfix, but never documented
+	log_deprecated(L,
+		"is_player_connected is undocumented and may be removed soon");
```

Comment:
`and will be removed in a future release`

`may` does not motivate enough for actions.